compose/ui/ui/src/desktopTest/kotlin/androidx/compose/ui/ExternalDragTest.kt

```diff
@@ -83,7 +86,6 @@ class ExternalDragTest {
             onDragInsideWindow(TestWindowDragValue(Offset(70f, 70f)))
         }
         awaitIdle()
-
         assertThat(events.size).isEqualTo(2)
         assertThat(events.last()).isEqualTo(Drag(Offset(70f, 70f)))
```

Comment:
This failed on Linux because of the large insets - the drag was outsude of the window

Reply:
I think better to not add the comment, as it adds unnessary explanation of non-important details, and we need to put it in every window test.

In every window test we ether need to use undecorated, or just use size big enough.